gcp/table_gcp_kubernetes_cluster.go

```diff
@@ -0,0 +1,424 @@
+				Description: "The authentication information for accessing the master endpoint.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "master_authorized_networks_config",
+				Description: "The configuration options for master authorized networks feature.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "network_config",
+				Description: "Configuration for cluster networking.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "network_policy",
+				Description: "Configuration options for the NetworkPolicy feature.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "node_config",
+				Description: "Parameters used in creating the cluster's nodes.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "node_pools",
+				Description: "The node pools associated with this cluster.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "notification_config",
+				Description: "Notification configuration of the cluster.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "private_cluster_config",
+				Description: "Configuration for private cluster.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "release_channel",
+				Description: "Release channel configuration.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "resource_labels",
+				Description: "The resource labels for the cluster to use to annotate any related Google Compute Engine resources.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "resource_usage_export_config",
+				Description: "Configuration for exporting resource usages.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "vertical_pod_autoscaling",
+				Description: "Cluster-level Vertical Pod Autoscaling configuration.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "workload_identity_config",
+				Description: "Configuration for the use of Kubernetes Service Accounts in GCP IAM policies.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "server_response",
+				Description: "ServerResponse contains the HTTP response code and headers from the server.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("googleapi.ServerResponse"),
+			},
+
+			// standard steampipe columns
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "akas",
+				Description: ColumnDescriptionAkas,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromP(gcpKubernetesClusterTurbotData, "Akas"),
+			},
+
+			// standard gcp columns
+			{
+				Name:        "location",
+				Description: ColumnDescriptionLocation,
+				Type:        proto.ColumnType_STRING,
+			},
```

Comment:
@rajeshbal65 Can you verify this  column value for `regional` and `zonal` cluster?
Response have both `location` and `zone` property. Do we requires both based on the cluster type, or `location` property gives data for both cluster location?

Reply:
![Screenshot 2021-05-24 at 1 39 16 PM](https://user-images.githubusercontent.com/46851617/119317227-7b513280-bc95-11eb-8308-785590ac7b25.png)

Reply:
![Screenshot 2021-05-24 at 1 40 16 PM](https://user-images.githubusercontent.com/46851617/119317354-9e7be200-bc95-11eb-888b-db2f1f2e3ed0.png)

Reply:
Location gives the region/zone of the cluster